src/types/api-error.d.ts

```diff
@@ -12,7 +12,7 @@
  */
 import { Headers } from 'node-fetch';
 
-export interface ErrorCause {
+export interface OktaApiErrorCause {
```

Comment:
Renamed cause latest spec has own `ErrorCause` model

Reply:
Renaming this would be a breaking change - are you able to avoid doing so?